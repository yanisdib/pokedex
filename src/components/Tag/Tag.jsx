import styled from 'styled-components';


export default styled.div`
    display: flex;
    width: fit-content;
    background-color: ${({ background }) => background};
    font-size: 0.85em;
    font-weight: 600;
    color: ${({ color }) => color};
    text-align: center;
    text-transform: uppercase;
    padding: 5px 15px;
    border-radius: 20px;
`;