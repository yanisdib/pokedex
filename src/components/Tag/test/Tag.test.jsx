import { shallow } from 'enzyme';

import Tag from '../Tag';


describe('Tag component', () => {
    let wrapper;

    beforeEach(() => wrapper = shallow(<Tag />));

    it('renders correctly without crashing', () => expect(wrapper).toMatchSnapshot())
})