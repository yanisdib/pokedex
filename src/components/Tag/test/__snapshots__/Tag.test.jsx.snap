// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag component renders correctly without crashing 1`] = `ShallowWrapper {}`;
